### ISSUE TITLE
High contrast scheme in debugger lacks contrast

### ISSUE THREAD

**Issue by BurtHarris** [Microsoft/vscode#28219]
- VSCode Version: 1.12.2
- OS Version: Windows 10

Thanks for your help with the last high-contrast issue, here's another.

Steps to Reproduce:

1.   Using the High Contrast color scheme
2.   Debug an application
3.   Step into a subroutine of some sort
4.   In the call stack window, select an earlier stack frame

Results in the code window show low contrast due to use of gray bar

 
![untitled](https://user-images.githubusercontent.com/2715647/26910080-9ca371c4-4bb9-11e7-817b-88d210cbe107.png)

**Comment by isidorn** [comment]
@BurtHarris these issues are great, please keep them coming. Here's the latest look which you will be able to see in the insiders in a couple of days

![screen shot 2017-06-08 at 15 31 44](https://user-images.githubusercontent.com/1926584/26931104-b36b85da-4c5f-11e7-935d-3265f2b4e95c.png)

**Comment by BurtHarris** [comment]
Thanks

**Comment by roblourens** [comment]
Here's what I see now @isidorn 

Top frame
![image](https://user-images.githubusercontent.com/323878/27665204-b2a952a0-5c20-11e7-8f61-202faedbb358.png)

Previous frame
![image](https://user-images.githubusercontent.com/323878/27665216-bcf214cc-5c20-11e7-9a2d-b19d44743c99.png)

**Comment by isidorn** [comment]
@roblourens good catch, thanks.

The issue was introduced because we stoped using blend mode in hc theme. Now I am doing the same as Alex in the editor for HC selection, I color all the tokens black when they are on the top frame / focused frame

**Comment by bpasero** [comment]
@isidorn looks good, though notice that inlined values might need some tweaking too: 

![image](https://user-images.githubusercontent.com/900690/27725530-090c819c-5d77-11e7-8af9-d2095ee7dab5.png)

**Comment by isidorn** [comment]
Thanks, created this one https://github.com/Microsoft/vscode/issues/29911 to track that
